addons/gamification/data/cron.xml

```diff
@@ -10,5 +10,42 @@
             <field name="numbercall">-1</field>
             <field eval="False" name="doall" />
         </record>
+
+        <record id="ir_cron_consolidate_last_day" model="ir.cron">
```

Comment:
I would remove the last day consolidation ... monthly / yearly should be sufficient. Even with a lot of students it is not an issue to keep karma gains.